### DISCUSSION TITLE
Italian qwerty keyboard

### DISCUSSION THREAD

**Discussion by tomtom-95** [monkeytypegame/monkeytype#3727] [Ideas] [answered]
Hi! 
I wanted to set the layout of the keyboard to Italian qwerty but it seems the option is not available.
Is it possible to add this layout?

**Comment by Miodec** [comment, marked as answer]
Can you share a link/image to what the layout should look like?

**Reply by tomtom-95** [reply]
![italian querty mac](https://user-images.githubusercontent.com/74787543/200552595-d9e075c8-c8af-4207-ae4a-3c530f5137f6.jpeg)

**Reply by Miodec** [reply]
Actually wait, i realised i can't copy the characters from an image haha. Can you find it in text (or type the characters for me).

**Reply by tomtom-95** [reply]
you mean you  want me to type the characters in the image above?

**Reply by Miodec** [reply]
Yeah, or ideally just find the layout in text. The reason is that i need to have the layout in a format like this: 
![image](https://user-images.githubusercontent.com/13181393/200554040-559751f5-4bc7-4f64-a7a1-631db9d7f9c7.png)

**Reply by tomtom-95** [reply]
“keys”: {
  “row1”: [“1!”, “2””, “3£”, “4$”, “5%”, “6&”, “7/”, “8(”, “9)”, “0=”, “‘?”, “ì^”],
  “row2”: [“qQ”, “wW”, “eE”, “rR”, “tT”, “yY”, “uU”, “iI”, “oO”, “pP”, “èé[”, “+*]”],
  “row3”: [“aA”, “sS”, “dD”, “fF”, “gG”, “hH”, “jJ”, “kK”, “lL”, “òç@”, “à°#”, “ù§”],
  “row4”: [“zZ”, “xX”, “cC”, “vV”, “bB”, “nN”, “mM”, “,;”, “.:”, “-_”],
  “row5”: []
}

is it ok?

**Reply by Miodec** [reply]
Should be ok, let me go add it
